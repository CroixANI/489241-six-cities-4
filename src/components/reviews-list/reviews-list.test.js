import React from 'react';
import renderer from 'react-test-renderer';

import ReviewsList from './reviews-list';
import OFFERS_TESTS from '../../mocks/offers-tests';

it(`Render Review component`, () => {
  const tree = renderer
    .create(<ReviewsList reviews={OFFERS_TESTS[0].reviews} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
