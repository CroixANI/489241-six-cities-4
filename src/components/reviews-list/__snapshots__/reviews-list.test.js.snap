// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Review List component Render Reviews List component for authenticated user 1`] = `
<section
  className="property__reviews reviews"
>
  <h2
    className="reviews__title"
  >
    Reviews · 
    <span
      className="reviews__amount"
    >
      0
    </span>
  </h2>
  <ul
    className="reviews__list"
  />
</section>
`;

exports[`Render Review List component Render Reviews List component for not authenticated user 1`] = `
<section
  className="property__reviews reviews"
>
  <h2
    className="reviews__title"
  >
    Reviews · 
    <span
      className="reviews__amount"
    >
      0
    </span>
  </h2>
  <ul
    className="reviews__list"
  />
</section>
`;
